CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Flutter Plugin Changelog
 
+## Version 5.0.0 - October 18, 2021
+Major release for Airship Android SDK 16.0.0 and iOS SDK 16.0.1.
+
+### Changes
+- Added Privacy Manager methods `enableFeatures`, `disableFeatures`, `setEnabledFeatures`, `getEnabledFeatures` and `isFeatureEnabled` that replace `getDataCollectionEnabled`, `setDataCollectionEnabled`, `getPushTokenRegistrationEnabled` and `setPushTokenRegistrationEnabled`
+- Support for OOTB Preference Center
+- Xcode 13 is now required.
```

Comment:
You should mention android new requirements too:
compile and target SDK version 31, java 8